bark/tools/dat-from-wav.py

```diff
@@ -0,0 +1,28 @@
+from scipy.io import wavfile as wavfile
+import bark, os
+
+
+def main(wav):
+	if not isinstance(wav, str):
+	    raise ValueError('A file path string is required')
+	name, ext = os.path.splitext()   # get the original file name. e.g. ab from ab.wav
+	rate, data = wavfile.read(wav)
+	# make dat file with the same name as wav file
+	datname = '{}.dat'.format(name)
+	bark.write_sampled(datname, data, rate, original_file=wav)
```

Comment:
If a user wanted to use this function independently of the command-line wrapper, it seems likely that they might want to be able to manipulate the resulting bark object after creating it. Why not return the object at the end of the function?